ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -234,15 +234,16 @@ class SecondaryNavBar extends React.PureComponent {
           </span>
           <form role="search" className="form-inline flex-row">
             {serverChanged && (
-              <span
+              <button
```

Comment:
We are transitioning towards using `react-strap` in our code base and I think the <Button> component will get this lint rule passing without having to add the `type` attribute.

Reply:
I looked at how it's implemented in the rest of this file, and there <span> used with `role="button"` and `tabindex="-1"` attribute. Maybe we should use it for consistency too?

Reply:
Well, so that's the thing to determine. It's preferable to use elements with semantic meaning rather than spans or divs with roles assigned to make them work like buttons. If you change it one place, you should definitely change it anywhere it makes sense. If it changes the appearance drastically you could take a screenshot and get an opinion from @camd

Reply:
Thank you. Fixed in my last commit. [Here](https://srv2.imgonline.com.ua/result_img/imgonline-com-ua-2to1-eAB6aDx81slZL4Op.png) is screenshot with before/after. From what I could see, the appearance isn't changed.